MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.kt

```diff
@@ -514,14 +537,24 @@ private class MuxDevice(ctx: Context) : IDevice {
     return "Android"
   }
 
-  override fun getMuxOSFamily(): String? = overwrittenOsFamilyName
+  override fun getMuxOSFamily(): String? {
+    return overwrittenOsFamilyName
+  }
 
   override fun getOSVersion(): String {
     return Build.VERSION.RELEASE + " (" + Build.VERSION.SDK_INT + ")"
   }
 
   override fun getMuxOSVersion(): String? = overwrittenOsVersion
 
+  override fun getDeviceName(): String = ""
+
+  override fun getMuxDeviceName(): String? = overwrittenDeviceName
+
+  override fun getDeviceCategory(): String = ""
```

Comment:
Is there a way we can detect this? It should be a value in: `phone`, `tablet`, `tv`, or `car`